staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured.go

```diff
@@ -80,6 +80,31 @@ func (obj *UnstructuredList) UnstructuredContent() map[string]interface{} {
 	return obj.Object
 }
 
+func (u Unstructured) DeepCopy() Unstructured {
+	return Unstructured{
+		Object: deepCopyJSON(u.Object).(map[string]interface{}),
+	}
+}
+
+func deepCopyJSON(x interface{}) interface{} {
```

Comment:
This is my understanding of the `Unstructured` data type (which resembles JSON).